File: web-services/services/auditing/src/main/java/com/cryptocaddy/services/auditing/model/Result.java
package com.cryptocaddy.services.auditing.model;

import java.util.ArrayList;
import java.util.List;

public class Result {
    private String status;
    private List<String> results;

    public Result(String status) {
        this.status = status;
        results = new ArrayList<>();
    }

    public String getStatus() {
        return status;
    }
    public List<String> getResults() {
        return results;
    }

    public void addResult(String resultToAdd){
        results.add(resultToAdd);
    }

}

File: web-services/auditing/src/main/java/com/cryptocaddy/services/auditing/api/dto/response/Result.kt
package com.cryptocaddy.services.auditing.api.dto.response

import java.util.ArrayList

class Result(val status: String) {
    private val results: MutableList<String>

    init {
        results = ArrayList()
    }

    fun getResults(): List<String> {
        return results
    }

    fun addResult(resultToAdd: String) {
        results.add(resultToAdd)
    }

}
